src/adr/0071-postgresql-mailbox-tables-structure.md

```diff
@@ -0,0 +1,44 @@
+# 71.  Postgresql Mailbox tables structure
+
+Date: 2023-12-14
+
+## Status
+
+Implemented
+
+## Context
+
+Apache James was integrated with Postgresql database. The table structure for mailbox was created to store mailbox data in Postgresql database.
+
+## Decision
+
+![diagram for mailbox tables](img/adr-71-mailbox-table-diagram.png)
```

Comment:
The diagram is nice but it do not replace plain english with a table list.

We are missing annotations on it, annotations is part of the mailbox.

I would also like to have the word **normalized** explicitly mentionned: our table structure is mostly normalized which mitigates storage costs and ease achiving consistency. Indexes are used for key query in order to improve performance.

Also have a word about constraintd

Reply:
fixed